Sources/Email/EmailAddress.swift

```diff
@@ -0,0 +1,48 @@
+import Vapor
+
+public struct EmailAddress: Content {
+    public let email: String
+    public let name: String?
+    
+    public init(email: String, name: String? = nil) {
+        self.email = email
+        self.name = name
+    }
+}
+
+extension EmailAddress: Emailable {
+    public static var emailKeyPath: KeyPath<EmailAddress, String> {
+        \.email
+    }
+    
+    public static var nameKeyPath: KeyPath<EmailAddress, String?>? {
+        \.name
+    }
+}
+
+extension EmailAddress: ExpressibleByStringLiteral {
```

Comment:
Is this something we even need to support? This is really only useful for the developer to use a string literal instead of the standard `EmailAddress` initializer, but it looks to me like something that's prone to bad formatting, causing much pain and heartache when trying to debug why on earth my `EmailAddress` instance doesn't contain the correct values.

Reply:
In my opinion, it is useful. To me this code: `VaporEmail(from: "test@test.com", "to: "test@test.com")` is cleaner than `VaporEmail(from: .init(email: "test@test.com"), to: email: "test@test.com")`. I get that there probably aren't a lot of people that would use the `Name <email@email.com>` format, but not sure what issues that would cause in terms of formatting. Open for more opinions, of course, could just be me finding it useful:) Thanks for feedback btw!